Main.java

```diff
@@ -9,7 +10,53 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+     // create a scanner for user input
+Scanner input = new Scanner(System.in);
+
+
+// ask the casier for the price 
+ System.out.println("Please enter the amount the customer spent");
+
+// get the discount price from the user
+double price = input.nextDouble();  
+
+double total = 0;
+double save = 0;
+// variable for discount for 0-40$ 10%
+double A =  price*.10;
+
+//variable for discount for $40.01 - $80.00
+double B = price*.20;
+
+//variable for discount for $80.01 - $120.00
+double C = price*.30;
+
+//variable for discount for over $120
+double D = price*.40;
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. Constants can be named with UPPERCASE letters, but need the word final out front of them.